packages/core/http/core-http-server-internal/src/lifecycle_handlers.test.ts

```diff
@@ -395,7 +395,13 @@ describe('customHeaders pre-response handler', () => {
   it('adds the kbn-name and Content-Security-Policy headers to the response', () => {
```

Comment:
__question:__ would you mind updating https://github.com/elastic/kibana/blob/7194ca267c8a1fd92e3f96a964993f8292e873c3/src/core/server/integration_tests/http/lifecycle.test.ts#L1477 as well to make sure we don't allow consumers to override `Content-Security-Policy-Report-Only`?